### PR TITLE
added joe to website

### PR DESCRIPTION
**Pull request by Peyton232**

### CODE REVIEW

File: about/our-team.php [hunk 1]
```diff
@@ -266,15 +266,15 @@
...
+                                <h6 class="h6-with-small"><a href="#"> Joe Moore</a><span class="small text-silver-chalice">Aerospace Division Lead</span></h6>
+                                <p>Joseph is a Junior Mechanical Engineering student. Formerly a US Marine Master Explosive Ordnance Disposal Technician (EOD), he has always tried to tinker with things and figure out how they work and is passionate about establishing and growing our High-Power Rocketry team into a nationally competitive force. Both through STEM outreach efforts and internal events, he hopes to generate interest and encourage everyone to get into hobby rocketry, and the sport of high power rocketry. </p>
                                 <ul class="list-inline-lg">
                                     <li>
-                                        <a href="mailto:aerospace@untrobotics.com " class="icon icon-sm text-primary fa-envelope"></a>
+                                        <a href="mailto:aerospace-division-lead@untrobotics.com " class="icon icon-sm text-primary fa-envelope"></a>
```
Comment (ntindle) [inline about/our-team.php:277]: Move Jesse to our-alumni page

**Review by ntindle — CHANGES_REQUESTED** [review]

**Review by sebastian-king — CHANGES_REQUESTED** [review]
The email is not aerospace-division-lead, it's just aerospace@

File: about/our-team.php (hunk 1 above)
Comment (ntindle) [reply about/our-team.php:277]: https://www.untrobotics.com/about/our-alumni

Comment (Peyton232) [reply about/our-team.php]: done

**Pull request merged**